qiskit/ignis/verification/randomized_benchmarking/dihedral.py

```diff
@@ -333,53 +338,53 @@ def __str__(self):
 
 class CNOTDihedral():
     """CNOT-dihedral Object Class.
-    The CNOT-dihedral group on n qubits is generated by the gates
+    The CNOT-dihedral group on num_qubits qubits is generated by the gates
     CNOT, T and X."""
 
-    def __init__(self, n_qubits):
-        # Construct the identity element on n_qubits qubits.
-        self.n_qubits = n_qubits
+    def __init__(self, num_qubits):
+        # Construct the identity element on num_qubits qubits.
+        self.num_qubits = num_qubits
         # phase polynomial
-        self.poly = SpecialPolynomial(n_qubits)
+        self.poly = SpecialPolynomial(num_qubits)
         # n x n invertible matrix over Z_2
-        self.linear = [[int(r == c) for c in range(n_qubits)]
-                       for r in range(n_qubits)]
+        self.linear = [[int(r == c) for c in range(num_qubits)]
+                       for r in range(num_qubits)]
         # binary shift, n coefficients in Z_2
-        self.shift = n_qubits * [0]
+        self.shift = num_qubits * [0]
 
     def _z2matmul(self, left, right):
         """Compute product of two n x n z2 matrices."""
-        prod = [[0 for col in range(self.n_qubits)]
-                for row in range(self.n_qubits)]
-        for i in range(self.n_qubits):
-            for j in range(self.n_qubits):
-                for k in range(self.n_qubits):
+        prod = [[0 for col in range(self.num_qubits)]
+                for row in range(self.num_qubits)]
+        for i in range(self.num_qubits):
+            for j in range(self.num_qubits):
+                for k in range(self.num_qubits):
                     prod[i][j] = (prod[i][j] +
                                   left[i][k]*right[k][j]) % 2
         return prod
 
     def _z2matvecmul(self, mat, vec):
         """Compute mat*vec of n x n z2 matrix and vector."""
-        prod = [0 for row in range(self.n_qubits)]
-        for i in range(self.n_qubits):
-            for j in range(self.n_qubits):
+        prod = [0 for row in range(self.num_qubits)]
+        for i in range(self.num_qubits):
+            for j in range(self.num_qubits):
                 prod[i] = (prod[i] + mat[i][j] * vec[j]) % 2
         return prod
 
     def __mul__(self, other):
         """Left multiplication self * other."""
-        assert self.n_qubits == other.n_qubits, "not same n_qubits!"
-        result = CNOTDihedral(self.n_qubits)
+        assert self.num_qubits == other.num_qubits, "not same num_qubits!"
+        result = CNOTDihedral(self.num_qubits)
         result.shift = [(x[0] + x[1]) % 2
                         for x in zip(self._z2matvecmul(self.linear,
```

Comment:
Actually if you can write `z2matmul` and `_z2matvecmul` these using `numpy.dot` you probably don't even need separate methods for them